packages/vitest/src/integrations/snapshot/manager.ts

```diff
@@ -70,7 +71,8 @@ export function addSnapshotResult(summary: SnapshotSummary, result: SnapshotResu
     })
   }
 
+  summary.skipped += result.skipped
```

Comment:
We don't care for skipped snapshots. We already show that a test was skipped, it should be enough. 

We shouldn't show a user anything about skipped tests.